sdk/formrecognizer/azure-ai-formrecognizer/tests/recordings/test_dmac_training.test_build_model_nested_schema_transform.yaml

```diff
@@ -48,36 +48,36 @@ interactions:
       Connection:
       - keep-alive
       User-Agent:
-      - azsdk-python-ai-formrecognizer/3.2.0b1 Python/3.9.0 (Windows-10-10.0.19041-SP0)
+      - azsdk-python-ai-formrecognizer/3.2.0b3 Python/3.8.5 (Windows-10-10.0.19041-SP0)
     method: GET
-    uri: https://region.api.cognitive.microsoft.com/formrecognizer/operations/31534702513_61c8c285-8e2a-4042-9b30-2beb4f33c349?api-version=2021-09-30-preview
+    uri: https://region.api.cognitive.microsoft.com/formrecognizer/operations/31527274843_83e52eec-d42a-48ad-a8ea-6a99b6449c32?api-version=2021-09-30-preview
   response:
     body:
-      string: '{"operationId": "31534702513_61c8c285-8e2a-4042-9b30-2beb4f33c349",
-        "kind": "documentModelBuild", "status": "succeeded", "createdDateTime": "2021-09-15T00:24:46Z",
-        "lastUpdatedDateTime": "2021-09-15T00:24:49Z", "resourceLocation": "https://region.api.cognitive.microsoft.com/formrecognizer/documentModels/4fa30281-9430-4b15-b99f-0c0f0f41ccec?api-version=2021-09-30-preview",
-        "percentCompleted": 100, "result": {"docTypes": {"4fa30281-9430-4b15-b99f-0c0f0f41ccec":
+      string: '{"operationId": "31527274843_83e52eec-d42a-48ad-a8ea-6a99b6449c32",
+        "kind": "documentModelBuild", "status": "succeeded", "createdDateTime": "2021-12-09T23:39:16Z",
+        "lastUpdatedDateTime": "2021-12-09T23:39:20Z", "resourceLocation": "https://region.api.cognitive.microsoft.com//formrecognizer/documentModels/4852f416-8369-4ac6-b5c5-4c676c2c4f1b?api-version=2021-09-30-preview",
```

Comment:
double-slash is interesting here. Hope it's just coming from the sanitizer.

Reply:
I think (hope) it's just the sanitizer, but I added an issue to investigate: #22156